File: MAANPP20/MAANPP20/MAANPP20/Models/Flights/Aeroplane.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MAANPP20.Models.Flights
{
    public class Aeroplane
    {
        [Key]
        public int id { get; set; }

        //[Required]
        public string name { get; set; }

        //[Required]
        public int numSeats { get; set; }

        public ICollection<AvioSediste> allSeats { get; set; }

        public bool deleted { get; set; }
    }
}

File: web2-Project/src/app/entities/aeroplane/aeroplane.ts
import { AvioSediste } from '../avio-sediste/avio-sediste';

export class Aeroplane {
    id: number; // id aviona
    name: string;
    numSeats: number; // broj mesta
    allSeats: Array<AvioSediste>; // lista sedista

    deleted: boolean;

    constructor(id: number, name: string, numSeats: number, allSeats: Array<AvioSediste>) {
        this.id = id;
        this.name = name;
        this.numSeats = numSeats;
        this.allSeats = allSeats;

        this.deleted = false;
    }
}
